src/module_library/c3CanAC.cpp

```diff
@@ -141,9 +141,10 @@ canopy_photosynthesis_outputs c3CanAC(
         // energy balance to get a better temperature estimate using that value
         // of stomatal conductance. Get the final estimate of stomatal
         // conductance using the new value of the leaf temperature.
-        double iabs_diff = light_profile.shaded_absorbed_ppfd[current_layer];  // micromole / m^2 /s
-        double pLeafshade = light_profile.shaded_fraction[current_layer];      // dimensionless
-        double Leafshade = LAIc * pLeafshade;                                  // dimensionless
+        double iabs_diff = light_profile.shaded_absorbed_ppfd[current_layer];    // micromole / m^2 /s
+        double j_diff = light_profile.sunlit_absorbed_shortwave[current_layer];  // J / m^2 / s
```

Comment:
I have not dug into the model itself, so I don' t know what's going on, but is this supposed to be `sunlit_absorbed_shortwave`?

Reply:
Whoops, this is a copy-paste mistake. It should be shaded.